src/test/java/core/basesyntax/handlers/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.handlers;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.storage.Storage;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static OperationHandler balanceOperationHandler;
+
+    @BeforeClass
+    public static void beforeClass() {
+        balanceOperationHandler = new BalanceOperationHandler();
+    }
+
+    @Test
+    public void balance_Ok() {
+        Storage.storage.put(new Fruit("mango"), 40);
+        balanceOperationHandler.process(new Fruit("mango"), 30);
+        Integer expected = 30;
+        assertEquals(expected, Storage.storage.get(new Fruit("mango")));
```

Comment:
```suggestion
        Integer expected = 30;
        Integer actual = Storage.storage.get(new Fruit("mango"));
        assertEquals(expected, actual);
```
давай зробимо так само в інших тестових методах (та в методах інших тестових класів хендлерів)